app/src/main/res/layout/activity_augmented_image.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".AugmentedImageActivity">
+
+
+    <fragment class="leo.me.la.codeblue.ar.AugmentedImageFragment"
```

Comment:
wrong field. class is not a valid one here
use `android:name="leo.me.la.codeblue.ar.AugmentedImageFragment"` instead